plugins/source/azure/resources/services/trafficmanager/profiles.go

```diff
@@ -0,0 +1,39 @@
+package trafficmanager
+
+import (
+	"context"
+
+	"github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/trafficmanager/armtrafficmanager"
+	"github.com/cloudquery/cloudquery/plugins/source/azure/client"
+	"github.com/cloudquery/plugin-sdk/v4/schema"
+	"github.com/cloudquery/plugin-sdk/v4/transformers"
+)
+
+func Profiles() *schema.Table {
+	return &schema.Table{
+		Name:                 "azure_trafficmanager_profiles",
+		Resolver:             fetchProfiles,
+		PostResourceResolver: client.LowercaseIDResolver,
+		Description:          "https://learn.microsoft.com/en-us/rest/api/trafficmanager/profiles/list-by-subscription?tabs=HTTP#profile",
+		Multiplex:            client.SubscriptionMultiplexRegisteredNamespace("azure_trafficmanager_profiles", client.Namespacemicrosoft_network),
```

Comment:
Nice, had to look up the namespace here https://learn.microsoft.com/en-us/azure/azure-resource-manager/management/azure-services-resource-providers 
![image](https://github.com/cloudquery/cloudquery/assets/26760571/b60fe250-50ae-47ad-9af3-799792752b69)

Reply:
Thanks, earlier, I was getting it from API Endpoint.